jobrunner/config.py

```diff
@@ -170,6 +173,25 @@ def parse_job_resource_weights(config_file):
 
 STATS_POLL_INTERVAL = float(os.environ.get("STATS_POLL_INTERVAL", "10"))
 
-
 # feature flag to enable new API abstraction
 EXECUTION_API = os.environ.get("EXECUTION_API", "false").lower() == "true"
+
+
+# k8s configurations:
+# 1 if want to run the k8s_runner in a local environment, e.g. minikube
+K8S_USE_LOCAL_CONFIG = os.environ.get("K8S_USE_LOCAL_CONFIG", "0") == 1
+
+# Storage class to be used, platform dependent
+K8S_STORAGE_CLASS = os.environ.get("K8S_STORAGE_CLASS", "standard")
+
+# Namespace to be used
+K8S_NAMESPACE = os.environ.get("K8S_NAMESPACE", "opensafely")
+
+# Location of the image of the job runner
+K8S_JOB_RUNNER_IMAGE = os.environ.get("K8S_JOB_RUNNER_IMAGE", "ghcr.io/opensafely-core/job-runner:latest")
+
+# Location of the image of the job runner
+K8S_STORAGE_SIZE = os.environ.get("K8S_STORAGE_SIZE", "20Gi")
+
+# The comma separated list of IP:PORT to be whitelisted by the execution job when `allow_database_access` is True
+K8S_EXECUTION_HOST_WHITELIST = os.environ.get("K8S_EXECUTION_HOST_WHITELIST", "")
```

Comment:
@chrisyeunggraphnet It would be good to make this more clearly backend-specific. @bloodearnest do you think there will be a config specific to the local Docker backend?

Reply:
Yes - the HIGH_PRIVACY_STORAGE_BASE and such.

Perhaps we could crudely namespace the env vars? eg. `GRAPHNET_*` and `LOCAL_*`?